charts/kubernetes-agent/Chart.yaml

```diff
@@ -12,8 +12,9 @@ dependencies:
   - name: kubernetes-monitor-chart
     version: "0.4.1"
     repository: "oci://docker.io/octopusdeploy"
-    condition: kubernetes-monitor-chart.enabled
+    condition: kubernetesMonitor.enabled
```

Comment:
Will update this chart reference once https://github.com/OctopusDeploy/lobster-watcher/pull/120 is merged